modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RankEvalTestHelper.java

```diff
@@ -42,4 +49,20 @@ public static XContentParser roundtrip(ToXContentToBytes testItem) throws IOExce
         XContentParser itemParser = XContentHelper.createParser(shuffled.bytes());
         return itemParser;
     }
+
+    public static void testHashCodeAndEquals(Object testItem, Object mutation, Object secondCopy) {
```

Comment:
Do we have this kind of helper method else where? If so should we re-use it? (Just nitpicking here, maybe add a TODO or open an issue to keep this in mind, no need to fix in this PR)